Makefile

```diff
@@ -0,0 +1,31 @@
+IMAGE_NAME?=jamhall/s3rver
+IMAGE_TAG?=latest
+
+.PHONY: lint
+lint:
+	npm run eslint
+	npm run prettier-check
+
+.PHONY: build
+build:
+	npm install
+
+.PHONY: test
+test:
+	npm run test
```

Comment:
Not quite sure I understand the purpose for why we need a `Makefile` for these cases. What is the benefit of running `make test` vs `npm run test`? Doesn't really seem like an advantage to me.

Reply:
At the risk of sounding unconstructive, I was also a little put off by the use of Makefile. I do think most of these would function fine as npm scripts as this is a relatively simple JavaScript library.

Reply:
I personally don't like have non-JavaScript things like Docker in `package.json`. `package.json` does not support variables what is useful for build image.

However, I am open to cooperation and taking into account the suggested changes. I will drop `Makefile` and create a new PR.